Analysis/Tasks/SkimmingTutorials/DataModel/LFDerived.h

```diff
@@ -23,9 +25,9 @@ namespace lftrack
 {
 DECLARE_SOA_INDEX_COLUMN(LFCollision, lfCollision);
 DECLARE_SOA_COLUMN(Pt, pt, float);
-DECLARE_SOA_COLUMN(P, p, float);
 DECLARE_SOA_COLUMN(Eta, eta, float);
 DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_DYNAMIC_COLUMN(P, p, [](float pt, float eta) { return pt * TMath::CosH(eta); });
```

Comment:
I merge this now, but please use math_utils:: in the next PR (see e.g. AnalysisDataModel.h)